downstream/modules/platform/proc-install-cli-aap-operator.adoc

```diff
@@ -55,40 +55,63 @@ spec:
   source: redhat-operators
   sourceNamespace: openshift-marketplace
 ---
-apiVersion: automationcontroller.ansible.com/v1beta1
-kind: AutomationController
-metadata:
-  name: example
-  namespace: ansible-automation-platform
-spec:
-  replicas: 1
-
 -----
 +
 This file creates a `Subscription` object called `_ansible-automation-platform_` that subscribes the `ansible-automation-platform` namespace to the `ansible-automation-platform-operator` operator.
 +
-It then creates an `AutomationController` object called `_example_` in the `ansible-automation-platform` namespace.
-+
-To change the {ControllerName} name from `_example_`, edit the _name_ field in the `kind: AutomationController` section of [filename]`sub.yaml` and replace `_<automation_controller_name>_` with the name you want to use:
-+
-[subs="+quotes"]
------
-apiVersion: automationcontroller.ansible.com/v1beta1
-kind: AutomationController
-metadata:
-  name: __<automation_controller_name>__
-  namespace: ansible-automation-platform
------
 . Run the [command]`*oc apply*` command to create the objects specified in the [filename]`sub.yaml` file:
 +
 -----
 oc apply -f sub.yaml
 -----
++
+. Verify the CSV PHASE reports Succeeded before proceeding using the [command]`oc get csv -n ansible-automation-platform` command:
```

Comment:
Should "Succeeded" be "succeeded" (capitalization)?

Reply:
I was mirroring it as it appears in the code block, so actually I should have used `Succeeded` I think actually, do you agree with that?

Reply:
Ah fair, I wonder if it's referencing the code block directly should it be denoted with quotes or something!?